src/EnvironmentChecker.php

```diff
@@ -4,17 +4,33 @@
 
 namespace Ecurring\WooEcurring;
 
-
 /**
  * Check if environment is suitable for this plugin to work.
  */
-class EnvironmentChecker {
+class EnvironmentChecker
+{
+    const MOLLIE_MINIMUM_VERSION = '6.0.0';
+
+    /**
+     * @return bool Whether Mollie plugin is active.
+     */
+    public function isMollieActive()
+    {
+        return is_plugin_active('mollie-payments-for-woocommerce/mollie-payments-for-woocommerce.php');
+    }
+
+    /**
+     * @return bool Whether current Mollie version is allowed
+     */
+    public function isMollieMinimalVersion(): bool
+    {
+        $molliePluginData = get_plugin_data(M4W_FILE);
```

Comment:
Probably it worth it to check if that constant exists? I guess it may cause problems in case of uncareful usage.

Reply:
Yup, good catch. thanks :)